index.bs

```diff
@@ -1798,6 +1822,66 @@ Note: Invoking this method from a [=browsing context=] where the [=Web Authentic
 
 </div>
 
+### Client Supported Capabilities (dictionary <dfn dictionary>SupportedCapabilities</dfn>) ### {#dictionary-supportedcapabilities}
+
+<xmp class="idl">
+    dictionary SupportedCapabilities {
+        boolean areU2FDevicesSupported;
+        boolean areCTAPDevicesSupported;
+        boolean areResidentKeysSupported;
+        boolean isUserVerificationSupported;
+        boolean isTokenBindingSupported;
+        UserVerifyingPlatformAuthenticatorAvailability userVerifyingPlatformAuthenticatorAvailability;
+        sequence<AuthenticatorTransport> transportsSupported;
+        sequence<DOMString> extensionsSupported;
+    };
+</xmp>
+
+This dictionary is used by [=[WRPS]=] to determine the supported capabilities of the [=client=].
+
+<div dfn-type="dict-member" dfn-for="SupportedCapabilities">
+    :   <dfn>areU2FDevicesSupported</dfn>
+    ::  This attribute indicates whether the [=client=] supports communicating with [[FIDO-U2F-Message-Formats]] devices.
+
+    :   <dfn>areCTAPDevicesSupported</dfn>
+    ::  This attribute indicates whether the [=client=] supports communicating with [[FIDO-CTAP]] devices.
+
+    :   <dfn>areResidentKeysSupported</dfn>
+    ::  This attribute indicates whether the [=client=] supports creating [=resident credentials=].
+
+    :   <dfn>isUserVerificationSupported</dfn>
+    ::  This attribute indicates whether the [=client=] supports [=user verification=].
+
+    :   <dfn>isTokenBindingSupported</dfn>
+    ::  This attribute indicates whether the [=client=] supports the [=Token Binding=] protocol.
+
+    :   <dfn>userVerifyingPlatformAuthenticatorAvailability</dfn>
+    ::  This attribute indicates the availability of a [=user-verifying platform authenticator=] on the [=client=].
+
+    :   <dfn>transportsSupported</dfn>
+    ::  This attribute indicates the {{AuthenticatorTransport}} values supported by the [=client=].
+
+    :   <dfn>extensionsSupported</dfn>
+    ::  This attribute indicates the [=extension identifiers=] supported by the [=client=].
+</div>
+
+### Capability Support - PublicKeyCredential's `getSupportedCapabilities()` Method ### {#sctn-getSupportedCapabilities}
+
+<div link-for-hint="WebAuthentication/getSupportedCapabilities">
```

Comment:
Will Microsoft's webauthn.dll export a reflection of this function for browsers running on >= 19H1? Some of the fields are browser specific, but some depend on the platform.

Reply:
Yes